konfera/models/talk.py

```diff
@@ -23,13 +24,20 @@
 
 class Talk(models.Model):
     title = models.CharField(max_length=256)
-    abstract = models.TextField()
+    abstract = models.TextField(help_text=_('Abstract will be published in the schedule.'))
     type = models.CharField(choices=TALK_TYPE, max_length=32)
     status = models.CharField(choices=TALK_STATUS, max_length=32)
-    duration = models.IntegerField(choices=TALK_DURATION)
+    duration = models.IntegerField(choices=TALK_DURATION, help_text=_('Talk duration in minutes.'))
     primary_speaker = models.ForeignKey('Speaker', related_name='secondary_speaker')
     secondary_speaker = models.ForeignKey('Speaker', related_name='primary_speaker', blank=True, null=True)
     event = models.ForeignKey('Event')
 
     def __str__(self):
         return self.title
+
+    def clean(self):
+        if self.primary_speaker == self.secondary_speaker:
```

Comment:
This validation doesn't seem to work as supposed.

If you don't select the primary speaker it throws `RelatedObjectDoesNotExist at /admin/konfera/talk/add/
Talk has no primary_speaker.` instead of validation error.